src/commands/util/user.js

```diff
@@ -1,14 +1,83 @@
-const { SlashCommandBuilder } = require("discord.js");
+const { SlashCommandBuilder, EmbedBuilder } = require("discord.js");
+const moment = require('moment');
+
 
 module.exports = {
 	data: new SlashCommandBuilder()
-		.setName("user")
-		.setDescription("Provides information about the user."),
+    .setName("userinfo")
+	.setDescription("Provides information about the user."),
+
 	async execute(interaction) {
-		// interaction.user is the object representing the User who ran the command
-		// interaction.member is the GuildMember object, which represents the user in the specific guild
-		await interaction.reply(
-			`This command was run by ${interaction.user.username}, who joined on ${interaction.member.joinedAt}.`
-		);
+		let guildMember;
+		let user = interaction.targetUser;
+
+		// Extract optional mentionable if it exists, otherwise get the user who called the interaction
+		if (user != null) {
+			guildMember = await interaction.guild.members.fetch(user);
+		} else {
+			user = interaction.user;
+			guildMember = interaction.member;
+		}
+
+		// Extract various data from the User and GuildMember Classes
+        const userRegistered = moment(user.createdAt).format('MMMM DD, YYYY');
+		const userName = user.username;
+		const userDiscrim = user.discriminator;
+		const userAvatar = user.displayAvatarURL();
+
+        const guildMemberJoined = moment(guildMember.joinedAt).format('MMMM DD, YYYY');
+		const guildMemberNickname = guildMember.nickname;
+		const guildMemberColor = guildMember.displayColor;
+
+        
+
+		let guildMemberPremium;
+		if (guildMember.premiumSince) {
+            guildMemberPremium = moment(guild.Member.premiumSince).format('MMMM DD, YYYY');
+		}
+
+
+		// Grab all the guild member's roles
+		const guildMemberRolesSize = guildMember.roles.cache.size - 1;
+		const guildMemberRoles = guildMember.roles.cache.sort((r1, r2) => r2.rawPosition - r1.rawPosition);
+		const guildMemberRolesText = guildMemberRoles
+			.map((r) => r)
+			.slice(0, -1)
+			.join(", ");
+
+		// Grab all the guild member's award roles
+		const guildMemberAwards = guildMember.roles.cache
```

Comment:
Remove awards logic since we do not use awards anymore

Reply:
Ok done.